app/assets/config/manifest.js

```diff
@@ -2,4 +2,5 @@
 //= link_tree ../builds
 //= link application.js
 //= link admin/domain-config.js
+//= link admin/modules/ga4-page-view-tracking.js
```

Comment:
should this not be in application.js? or is there an issue with it loading order?